go/example/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+var host = flag.String("host", ":3423", "host address:port")
+
+type AgentInfo struct {
+	Owner  string
+	Taunts []string
```

Comment:
the taunts are cute but seem fairly pointless, since almost all games will be AI against AI so it's like walls shouting at walls. how much do you want to keep them in?